packages/metro-resolver-symlinks/test/__fixtures__/remap-platforms/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "@rnx-kit/metro-resolver-symlinks/remap-tests",
+  "version": "0.0.0-dev",
+  "dependencies": {
+    "react-native": "0.0.0-dev",
+    "react-native-macos": "0.0.0-dev",
+    "react-native-win32": "0.0.0-dev",
```

```suggestion
    "@office-iss/react-native-win32": "0.0.0-dev",
```

This package is published under the `@office-iss` scope.